Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -152,6 +187,11 @@ GrDirectContext* PlatformDisplay::skiaGrContext()
     return s_skiaGLContext->skiaGrContext();
 }
 
+int PlatformDisplay::MSAASampleCount()
```

Comment:
I think we should make sure that this information is included in webkit://gpu.

Reply:
I haven't added this cause it means adding skia to the UI process, which I'm not sure it's a good idea.

Reply:
We are already using skia in the ui process, but I think we could add any skia info to webkit://gpu in a different patch.